HW8/Game/src/Character.java

```diff
@@ -0,0 +1,57 @@
+public class Character {
+    String name;
+    int health;
+    int strength;
+    int defence;
+    static int CHARACTER_COUNT;
```

Comment:
It's static, which serves the purpose, but the variable naming `ALL_CAPS` is used for constants in Java.
In our case the value is getting changed (incremented in the constructor method, so the regular `static int characterCount` would be a better choice

`static final int SOME_CONSTANT = 10`